certified-connectors/Cyberday/apiDefinition.swagger.json

```diff
@@ -0,0 +1,370 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Cyberday",
+    "description": "Cyberday is an information security management system, where one key part is managing the data assets of a company. Often the most important data asset are the data systems used to process data and run operations. Sometimes organizations already have some other locations for asset listings. This connectors helps to bring info of all assets to Cyberday, so the asset information and the decided safeguards to deploy can live in the same place.",
```

Comment:
All descriptions, titles, summaries, etc. must be free from grammar and spelling errors ("This connectors" => "This connector"). Please review them and update them accordingly.